bigquery_etl/cli/backfill.py

```diff
@@ -343,35 +343,43 @@ def scheduled(ctx, qualified_table_name, sql_dir, project_id, status, json_path=
     ConfigLoader.get("default", "project", fallback="moz-fx-data-shared-prod")
 )
 @click.pass_context
-def process(ctx, qualified_table_name, sql_dir, project_id):
-    """Process backfill entry with drafting status in backfill.yaml file(s)."""
-    click.echo("Backfill processing initiated....")
+def initiate(ctx, qualified_table_name, sql_dir, project_id):
+    """Process backfill entry with initiate status in backfill.yaml file(s)."""
+    click.echo("Backfill processing (initiate) started....")
 
     backfills_to_process_dict = get_scheduled_backfills(
-        sql_dir, project_id, qualified_table_name
+        sql_dir, project_id, qualified_table_name, status="Initiate"
     )
 
     if backfills_to_process_dict:
-        entry_to_process = backfills_to_process_dict[qualified_table_name]
+        entry_to_initiate = backfills_to_process_dict[qualified_table_name]
 
-        click.echo(f"\nValidating backfill for {qualified_table_name} via dry run:")
-        _process_backfill(ctx, qualified_table_name, entry_to_process, dry_run=True)
+        click.echo(
+            f"\nInitiating backfill(s) for {qualified_table_name} with entry date {entry_to_initiate.entry_date} via dry run:"
+        )
+        _initiate_backfill(ctx, qualified_table_name, entry_to_initiate, dry_run=True)
```

Comment:
A potential issue here: Some of these dry runs could take a while, potentially longer than an hour. The dry run also happens before the actual processing starts, which creates the staging table. If the next run starts before the dry run ends, `get_scheduled_backfills` won't see the staging table and return the backfill twice so we'll have two backfills for the same table being initiated concurrently. 

Some options I can think of:
1. Create the staging table immediately and if the dry run fails, delete it. 
2. Choose some other method of validating the backfill besides dry run
3. Limit the initiate DAG run to one max active run

Wdyt?

Reply:
Sounds good